### DISCUSSION TITLE
Generate delete.yaml

### DISCUSSION THREAD

**Discussion by robertjahn** [Dynatrace/dynatrace-configuration-as-code#239] [Archive]
I am coming from the view point of using monaco for demo setup, where I want to run monaco to setup my demo then remove all the config related to the demo setup once my demo is over.

So have all my config setup the way I like it, and it would be nice to say something like `monaco generate-delete-file —project xxx --filename delete.yaml` as to get the `delete.yaml` file that can then keep for my project.

**Comment by tatomir146** [comment]
Discussion in #218
